apps/sms/test/marionette/lib/views/dialer.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var View = require('./view');
+var Dialer =  require('../../../../../../apps/communications/dialer/test/marionette/lib/dialer.js');
+
+function DialerWrapper(client) {
+  this.client = client;
+}
+module.exports = DialerWrapper;
+
+DialerWrapper.prototype = {
+  __proto__: View.prototype,
+
+  switchToFrame: function() {
```

Comment:
I see that ```switchToFrame``` _without arguments_ in marionette js means switch to top-level frame, so here it looks a bit confusing...